src/gsAssembler/gsExprHelper.h

```diff
@@ -136,7 +136,7 @@ class gsExprHelper
         m_vlist.push_back( expr::gsFeVariable<T>() );
         expr::gsFeVariable<T> & var = m_vlist.back();
         gsFuncData<T> & fd = m_ptable[&mp];
-        fd.dim = mp.dimensions();
+        //fd.dim = mp.dimensions();
```

Comment:
Not sure if this is correct. I mean, I said that gcc 4.8.x fails at this point (or better inside dimensions(), where the linker can't find the correct implementation for targetDim()), but I didn't meant that it should be kicked out.

Reply:
I took it out for other reasons, it is not needed here..